veryfi/errors.py

```diff
@@ -1,22 +1,16 @@
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import unicode_literals
-
-
 class VeryfiClientError(Exception):
-    required_fields = []
```

Comment:
I don't see where it's used, so I cleaned it.

Reply:
Great, thank you!